# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/newtab/jar.mn
# Commit: 626dd2a045af
# Full Hash: 626dd2a045af6cd510ef04e67d027909509547f1
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2020-04-17 21:46:50
# Regressor Bug: 1614502
# File Overlap Count: 2
# Description:
#   Bug 1614502 - Add a mechanism that can generate an about:home document for a startup cache. r=Gijs,k88hudson
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D68094
# ==============================================================================

diff -r 7ec776d31a2c -r 626dd2a045af browser/components/newtab/jar.mn
--- a/browser/components/newtab/jar.mn	Fri Apr 17 15:31:11 2020 +0000
+++ b/browser/components/newtab/jar.mn	Fri Apr 17 15:31:39 2020 +0000
@@ -20,6 +20,8 @@
   res/activity-stream/vendor/react-transition-group.js (./vendor/react-transition-group.js)
   res/activity-stream/vendor/redux.js (./vendor/redux.js)
   res/activity-stream/vendor/react-redux.js (./vendor/react-redux.js)
+* res/activity-stream/data/content/abouthomecache/page.html.template (./data/content/abouthomecache/page.html.template)
+* res/activity-stream/data/content/abouthomecache/script.js.template (./data/content/abouthomecache/script.js.template)
   res/activity-stream/data/content/assets/ (./data/content/assets/*)
   res/activity-stream/data/content/tippytop/ (./data/content/tippytop/*)
   res/activity-stream/data/content/activity-stream.bundle.js (./data/content/activity-stream.bundle.js)